dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphIntegrationTest.java

```diff
@@ -97,13 +106,15 @@ public void testJaxen() throws RepositoryException {
     DefaultArtifact jaxen =
         new DefaultArtifact("jaxen:jaxen:1.1.6");
     DependencyGraph graph =
-        dependencyGraphBuilder.buildCompleteGraph(new Dependency(jaxen, "compile")).getDependencyGraph();
+        dependencyGraphBuilder
+            .buildFullDependencyGraph(ImmutableList.of(jaxen))
+            .getDependencyGraph();
 
     List<Update> updates = graph.findUpdates();
-    Truth.assertThat(updates).hasSize(3);
+    Truth.assertThat(updates).hasSize(6);
 
     List<DependencyPath> conflicts = graph.findConflicts();
-    Truth.assertThat(conflicts).hasSize(5);
+    Truth.assertThat(conflicts).hasSize(10);
```

Comment:
Because now the graph includes provided dependencies, the number of calculated conflicts increases. This will affect the dependency convergence column.